src/coreclr/tools/aot/Mono.Linker.Tests.Cases.Expectations/Assertions/KeptAttribute.cs

```diff
@@ -8,5 +8,11 @@ namespace Mono.Linker.Tests.Cases.Expectations.Assertions
 	[AttributeUsage (AttributeTargets.All, Inherited = false)]
 	public class KeptAttribute : BaseExpectedLinkedBehaviorAttribute
 	{
+		/// <summary>
+		/// By default the target should be kept by all platforms
+		/// This property can override that by setting only the platforms
+		/// which are expected to keep the target.
+		/// </summary>
+		public ProducedBy KeptBy { get; set; } = ProducedBy.TrimmerAnalyzerAndNativeAot;
```

Comment:
nit: kind of weird name to repeat the name of the attribute on the property.

Reply:
So just `[Kept (By = Tool.Trimmer)]`? (After we rename the enum...)
I can change that, sounds good.